### PR TITLE
[16.0][MIG] account_reporting_weight: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa  TT42419

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
@sergio-teruel  ping, @pedrobaeza

File: account_reporting_weight/i18n/account_reporting_weight.pot [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 13.0\n"
+"Project-Id-Version: Odoo Server 15.0\n"
```
Comment (pedrobaeza) [inline account_reporting_weight/i18n/account_reporting_weight.pot]: Why this change in your commit?

File: account_reporting_weight/static/description/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
```
Comment (pedrobaeza) [inline account_reporting_weight/static/description/index.html]: Why this change in your commit?

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]
Code and functional review.

**Comment by sergio-teruel** [comment]
ping @juangomezzurita
